docs/error-reporting/project-setup/symbolication.md

```diff
@@ -72,162 +67,128 @@ Backtrace supports the dSYM format.
+- Proxy Options - If a proxy server is required, these options define the proxy credentials, host, and port.
+- Download Options - How many concurrent downloads to allow, and retry options before adding a symbol to the skip list.
 
-    On Windows, the executable, libraries, and PDB files must be uploaded as some unwinding information required for symbolication is contained exclusively in the executable objects.
+### Managing symbol servers
 
-- .dsym files - MacOS .dsym files are supported.
-- .elf files - ELF files are supported.
+The Symbol Servers Management UI is accessed under Project Settings > Symbols > Symbol Servers. Following is some brief information about each of the tabs and the data within them.
+- **Strongly recommended**: Allowlist - If the allowlist is enabled, then only symbol files in the allowlist will be downloaded. For the public symbol servers that are seeded by default, allowlists are enabled and commonly used symbols are listed for retrieval.
+- Statistics and Usage - This tab show information about how much has been data has been downloaded, and how many successful or failed downloads there have been since the Symbol Server was added to the system
+- Blocklist - Items in the blocklist will never be downloaded from the symbol server. Add items to the blocklist if they will not change stack trace information, will not add any additional debugging information, or might cause only networking problems (i.e files too big, they change too often, etc.).
+- Skip list - Items are automatically added to the skip list if they cannot be downloaded from the symbol server in the specified number of retries (default is 3). Symbols in the skip list will not be requested. The number of retries can be configured. Skip list items over 12 hours old are flushed.
+- Logs - Log information about successful and unsuccessful download and conversion attempts.
 
-Once you locate and upload missing symbols, make sure to Reprocess Objects, as described in the previous section on missing symbols.
-
-### Mismatched Module Name
-
-In addition to not having uploaded the proper symbol file, another cause for mixups is having a mismatched module name. When viewing missing symbol information, Backtrace will show the name of the module name (executable or dll/library file) that it's expecting that symbol file to match. This is "path" in the missing symbols section of annotations in the Debugger, or shown as "Library" when looking at the tooltip of a frame that's shown as missing symbols.
+## Troubleshooting
 
-If the name of the module doesn't match, Backtrace will show the symbol is missing, so make sure your symbol files reflect the correct module name, and if uploading exe/dll files, make sure these have the correct names.
+If you are blocked on uploading symbols from the command line, try uploading directly from your web browser. Instructions are available above.
```

Comment:
Could this topic have a header, like there is an ### Invalid Token or ### Missing Symbols headers for the next two topics?